File: BankApp/src/app/Modules/AuditLog.ts
import { Employee } from "./Employee";

export interface AuditLog {
    auditLogId?: number | null;
    action?: string | null;
    employeeId?: string | null;
    actionDate?: Date | null;
    ipAddress?: string | null;
    details?: string | null;
    employee?: Employee | null;
}

File: BankApplicationAPI/BankApplicationAPI/Models/AuditLog.cs

namespace BankApplicationAPI.Models;

public partial class AuditLog
{
    public int? AuditLogId { get; set; }

    public string? Action { get; set; } = null!;

    public string? EmployeeId { get; set; } = null!;

    public DateTime? ActionDate { get; set; }

    public string? IpAddress { get; set; }

    public string? Details { get; set; }
    
    public virtual Employee? Employee { get; set; } = null!;
}
